### DISCUSSION TITLE
My wireless headphones have run out of charge and then game crashed.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13606] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

It seems to me that if the audio device is turned off the game should not to crash?

I was playing network game and was the server host.
The server hosted from in-game menu was on the same computer where my client was.
I played using wireless bluetooth headphones with a microphone. My wireless headphones have run out of charge. I opened the settings and changed the playback device to USB headphones. I didn’t think to change the microphone device. After that I turned off my bluetooth headphones. The game crashed. Two hours of gameplay was wasted.

[servercrashreport (9).log](https://github.com/FakeFishGames/Barotrauma/files/14620780/servercrashreport.9.log)
[servercrashreport (10).log](https://github.com/FakeFishGames/Barotrauma/files/14620782/servercrashreport.10.log)
[crashreport (3).log](https://github.com/FakeFishGames/Barotrauma/files/14620783/crashreport.3.log)

### Reproduction steps

1. Use wireless bluetooth headphones to listen game sounds.
2. Start server and start round.
3. Turn off wireless bluetooth headphones.

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Hello and thank you for your report.

Tried to reproduce this issue and it might have been fixed in one of the recent audio device fixes.
I got this notification in debug console when turning off headphones
<img width="606" height="183" alt="image" src="https://github.com/user-attachments/assets/df91dc37-245e-42e0-9325-67b4963ee3d2" />

Closing discussion for now
